meeting_minutes/250109_SARIF_TC_95.md

```diff
@@ -0,0 +1,157 @@
+
+| First Name | Last Name     | Company                 | Role(s)                 | Present |
+|:-----------|:--------------|:------------------------|:------------------------|:--------|
+| Aditya     | Sharad        | Microsoft               | Voting Member           | ???     |
+| Alexandre  | Dulaunoy      | CIRCL                   | Member                  | ??      |
+| Andras     | Iklody        | CIRCL                   | Member                  | ??      |
+| Arjun      | Gopalakrishna | Microsoft               | Voting Member           | ??      |
+| Charles    | Wilson        | Torc Robotics, Inc.     | Member                  | ???     |
+| Chris      | Meyer         | Microsoft               | Member                  | ??      |
+| David      | Keaton        | Individual              | Co-Chair                | ???     |
+| David      | Malcolm       | Red Hat                 | Voting Member           | ???     |
+| Eddy       | Nakamura      | Microsoft               | Member                  | ??      |
+| Kevin      | Greene        | Mitre Corporation       | Member                  | ??      |
+| Luke       | Cartey        | Microsoft               | Co-Chair                | ??      |
+| Mary       | Martin        | Microsoft               | Member                  | ??      |
+| Michael    | Fanning       | Microsoft               | Member                  | ??      |
+| Michael    | Omokoh        | Microsoft               | Voting Member           | ???     |
+| Nathan     | Baird         | Microsoft               | Voting Member           | ??      |
+| Paul       | Brookes       | Microsoft               | Member                  | ??      |
+| Paul       | Seay          | Northrop Grumman        | Member                  | ??      |
+| Ross       | Wollman       | Microsoft               | Member                  | ??      |
+| Stefan     | Hagen         | Individual              | Secretary, taking notes | ???     |
+| Thanassis  | Avgerinos     | ForAllSecure Inc        | Voting Member           | ???     |
+| Tim        | Hudson        | Cryptsoft Pty Ltd.      | Member                  | ??      |
+| Vadim      | Okun          | NIST                    | Observer                | ??      |
+
+... of the nine voting members present - quorum was ... reached.
+
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* [Agenda for January 9, 2025](https://www.oasis-open.org/committees/download.php/number/)
+
+The agenda was ...
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2024-12-12 Meeting #94](https://www.oasis-open.org/committees/download.php/number/)
+
+The minutes were ...
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+* ACTION on Aditya to provide an informal text to guide escaping of URIs
+  * ...
+* ACTION on David Malcolm to comment on #588 sketching a proposal for #line directives
+  * ...
```

Comment:
Should the above issue link be #618 or even #572 rather than #588?
It seems like a paste-o from the action item below it. Otherwise it isn't clear to me what the relation between diagrams and line directives might be.

Reply:
Thanks; yes this does look like it should be #618.  FWIW I've just added a long comment there with some simple concrete examples, but I don't yet have a proposal.